### PR TITLE
Testing part1

### PR DESCRIPTION
**Pull request by herokudev**
Create testing for the add and delete functions.

### CODE REVIEW

File: src/spec/ListOfTasks.test.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import ListOfTasks from '../task-list.js';
```
Comment (Dipeshtwis) [inline src/spec/ListOfTasks.test.js:5]: - [ ] Make sure that tests are passing. I have got the CLI error as well. Please fix that. Looks like there is a parsing error, if in your system works fine then please send a screenshot as well to confirm it.

Screenshot from 2021-09-02 01-57-28


Screenshot from 2021-09-02 01-54-55

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @herokudev and @petumazo, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: Your Linters are passing :white_check_mark: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

- [x] I would recommend adding the command in README on how to test the project. [Hint: `npm test`]


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/spec/ListOfTasks.test.js (hunk 1 above)
Comment (herokudev) [reply src/spec/ListOfTasks.test.js:5]: All test on green on local system as show on screenshot.png

Comment (sinansevgi) [reply src/spec/ListOfTasks.test.js:5]: I ran the test on my local environment and all passed :confetti_ball: 
Screenshot from 2021-09-02 01-01-00

**Review by sinansevgi — APPROVED** [review]
# Approved :heavy_check_mark: 

Hi @herokudev and @petumazo , 

All tests are passing :clap: 
Your project is complete!  
Congratulations! 🎉

You can merge this pull request and continue with your journey :wave: 


Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**